cmd/keytransparency-client/grpcc/grpc_client.go

```diff
@@ -139,7 +139,11 @@ func min(x, y int32) int32 {
 
 // ListHistory returns a list of profiles starting and ending at given epochs.
 // It also filters out all identical consecutive profiles.
+// Epochs start at 1.
 func (c *Client) ListHistory(ctx context.Context, userID, appID string, start, end int64, opts ...grpc.CallOption) (map[*trillian.SignedMapRoot][]byte, error) {
+	if start <= 0 {
```

Comment:
This is the client with parameters the caller provides (could be any `int64`) and not by a recieved proto. Hence, I think it makes sense to keep it like this. We could change the params to unsigned `uint64` instead (than the check `==0` would be right).